data/os/RedHat-7.yaml

```diff
@@ -10,6 +10,9 @@ simp::server::scenario_map:
   poss: "%{alias('simp::scenario::data::el7')}"
   simp: "%{alias('simp::scenario::data::el7')}"
   simp_lite: "%{alias('simp::scenario::data::el7')}"
+
+# Most EL7 systems point at OpenLDAP and we don't want to cause user issues
+simp::sssd::client::ldap_389ds_compat: false
```

Comment:
I would default all the clients to the same setting since the compatibility  depends on the server and not the client OS.